bots/pr/src/main/java/org/openjdk/skara/bots/pr/CheckRun.java

```diff
@@ -885,6 +886,12 @@ private void checkStatus() {
                 rebasePossible = false;
             }
 
+            var original = backportedFrom();
+            var isCleanBackport = false;
+            if (original.isPresent()) {
+                isCleanBackport = updateClean(original.get());
```

Comment:
This looks fine as long as there is no problem calling `backportedFrom()` and `updateClean()` earlier than was done previously.

Reply:
Indeed. I have checked this as well as I can and the tests are passing. I can't see any issue with running it before.